src/extractor/scripting_environment_lua.cpp

```diff
@@ -284,17 +284,26 @@ void Sol2ScriptingEnvironment::InitContext(LuaScriptingContext &context)
         [&context, &get_location_tag](const osmium::Node &node, const char *key) {
             return get_location_tag(context, node.location(), key);
         });
-
+    // just for backward compatibility
+    // TODO: remove in v6
     context.state.new_enum("traffic_lights",
```

Comment:
I propose to use common `traffic_flow_control_direction` enum instead of existing `traffic_lights`